### PR TITLE
cpu: integrate sbchl and adchl instructions

### PR DESCRIPTION
**Pull request by toastbucket**

### CODE REVIEW

File: src/cpu/mod.rs [hunk 1]
```diff
@@ -14,12 +14,12 @@ enum Instruction {
     AddImm(),
     AddHL(),
     Adc(ArithmeticOperand),
-    // TODO: AdcHL
+    AdcHL(),
```
Comment (toastbucket) [inline src/cpu/mod.rs:17]: Maybe we wanna change the name of these to match Ld.  If we go with LdFromMem maybe AdcFromMem or something.

Comment (ricfehr3) [reply src/cpu/mod.rs:17]: I'd hate to admit it, but when you're right you're right. Go ahead. The variable name must not be an obstacle.

Comment (toastbucket) [reply src/cpu/mod.rs]: Added 3 new commits to address this.  You know me, all or nothin baby.

**Comment by ricfehr3** [comment]
+420 and I like it. Merged.

**Pull request merged**